src/naming.jl

```diff
@@ -250,3 +250,89 @@ end
 function dict2ntuple(dict::Dict{Symbol, T}) where T
     NamedTuple{Tuple(keys(dict))}(values(dict))
 end
+
+
+"""
+    parse_savename(filename::AbstractString; kwargs...)
+                        parsetypes = (Int, Float64),
+                        connector = "_")
```

Comment:
I assume that the extra keywords here were forgotten in the call signature, since you very nicely explain them later on in their own section.

Reply:
Oh yes. I usually have them in the call signature, but wanted to match your style